### ISSUE TITLE
Deprecate add_ruleset and add_ruleset_string

### ISSUE THREAD

**Issue by p0pr0ck5** [p0pr0ck5/lua-resty-waf#263]
Need to start a deprecation cycle for reading JSON data and moving over exclusively to SecRules consumption.

**Comment by p0pr0ck5** [comment]
Slating deprecation for v0.12. Will remove in v1.0.

**Comment by tzumby** [comment]
If I'm understanding this correctly, with this you load just one ruleset, right ? Something like REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf from the modsecurity rules. 

lua_resty_waf.load_secrules("/path/to/secrules/ruleset_name")

How would waf:set_option("add_ruleset", {"rule1.conf", "rule2.conf}) work when you pass a table? Would you have multiple load_secrules for the different rules ?

**Comment by p0pr0ck5** [comment]
Yeah, you can just call load_secrules multiple times. Or for convenience we can allow this to take a table, that's really
Just syntactic sugar :)
